modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1713,6 +1713,10 @@
 			<description name="Reorder parameters in the getItems and getItemsCount methods of the FDSDataProvider interface" />
 			<weight>1</weight>
 		</check>
+		<check name="UpgradeJavaGetCategoryIdMethodCheck">
+			<category name="Upgrade" />
+			<description name="Replace the getLeftCategoryId method with getCategoryId" />
```

Comment:
@MicaelleSilvaa, From which classes? We need to tell.

Reply:
Resolved